runtime/gc_base/GCExtensions.hpp

```diff
@@ -107,6 +107,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	bool collectStringConstants;
 
 	MM_MarkJavaStats markJavaStats;
+	MM_ContinuationStats continuationStats;
```

Comment:
Minor, but I'd put it just after any collector specific stats (so after Scavenger stats). Same for Environment and and other code that deals with it (merge, clear, etc)

Reply:
Also we should probalby  put this under JAVA > 19

Now that I mentioned it, I remember we did not do it for Continuation lists, and we probably should (but in a separate change)